doc/meetings/2024-09-12_Minutes.md

```diff
@@ -0,0 +1,88 @@
+---
+title:  "[Reproducible research and collaboration in git + Github]: Workshop preparation meeting"
+date:   "2024-09-12"
+output: github_document
+---
+
+**Attendees:**
+
+-   Daniel Morillo-Cuadrado
+    [dvmorillo\@psi.uned.es\>](mailto:dvmorillo@psi.uned.es){.email} [DMC]
+
+-   Kevin Bairos-Novak
+    [kbairosnovak\@gmail.com](mailto:kbairosnovak@gmail.com){.email} [KBN]
+
+**Scribe:** DMC
+
+**Schedule:** 09:00 UTC - 10:00 UTC
+
+# Action points
+
+## Decide workshop approach
+
+### Discussion
+
+-   Proposed instructor roles: DMC facilitator, JBN co-facilitator (moderation,
+    technical assistance, comments)
+
+### Agreements
+
+-   **Approved**: 8 participants (ideal), 10 hard limit (if high demand)
+
+-   **Approved**: Agreement on proposed instructor roles
+
+### Tasks
+
+-   **DMC**: Find out about split rooms [ASAP]
+
+-   **KBN**: Propose system for providing technical assistance [09-30]
+
+-   **DMC**: Add technical assistance info to slides [10-02]
+
+## Decide contents
+
+### Discussion
+
+-   KBN proposes to test the slides with lab colleagues
+
+### Tasks
+
+-   **KBN**: Send DMC UQ logo [ASAP]
+
+-   **DMC**: Update contents [09-20]
+
+-   **KBN**: Test contents [09-30]
+
+## Schedule next meeting
+
+### Discussion
+
+-   October 8 the latest according to schedule (best before October 7)
+
+### Agreements
+
+-   **Approved**: Set on October 3, same schedule
+
+# Next meeting
+
+-   **Date**: 2024-10-03
+
+-   **Schedule**: 09:00 UTC - 10:00 UTC
+
+# **Task summary:**
+
+## DMC
+
+-   Find out about split rooms [ASAP]
+
+-   Add technical assistance info to slides [10-02]
+
+-   Update contents [09-20]
+
+## KBN
+
+-   Propose system for providing technical assistance [09-30]
+
+-   Send DMC UQ logo [ASAP]
+
+-   Test contents [09-30]
```

Comment:
Done! Modified R code a little and one slide was going off the page (R environment) so divided into two

Reply:
Cool! I'll have a look in that PR ;)